src/v2/Components/FollowButton/FollowArtistButton.tsx

```diff
@@ -13,19 +13,22 @@ import React from "react"
 import track, { TrackingProp } from "react-tracking"
 import styled from "styled-components"
 import { FollowArtistButton_artist } from "../../__generated__/FollowArtistButton_artist.graphql"
+import { FollowArtistButtonQuery } from "../../__generated__/FollowArtistButtonQuery.graphql"
 import { FollowButton } from "./Button"
 import {
   RelayProp,
   commitMutation,
   createFragmentContainer,
   graphql,
+  QueryRenderer,
```

Comment:
We'll want to be using `SystemQueryRenderer` so as to guard against the extra unnecessary req during the SSR pass

Reply:
Nice